CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## v0.6.1 (Unreleased)
```

Comment:
I forgot to mention: since this is a breaking change, we should bump it to `0.7.0`.